doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -178,6 +178,8 @@ nRF9160: Asset Tracker v2
 * Updated:
 
   * Replaced deprecated LwM2M API calls with calls to new functions.
+  * Removed static modem data handling from the application's nRF Cloud codec.
+    Enabled the :kconfig:option:`CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS` configuration option to send static modem data.
```

Comment:
You actually enable `CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS_SIM`.  Which is correct?

Reply:
both, CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS is default y

Reply:
OK.  I got confused because of this change in prj.conf at the same time:
```
-CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS=n
+CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS_SIM=y